src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -457,12 +457,12 @@ class TableToolPage extends Component<Props, State> {
                                 <>
                                   <div>Generated permanent link:</div>
                                   <LinkContainer
-                                    url={`${window.location.href}/permalink/${permalinkId}`}
+                                    url={`${window.location.host}/data-tables/permalink/${permalinkId}`}
                                   />
                                   <div>
                                     <a
                                       className="govuk-link"
-                                      href={`${window.location.href}/permalink/${permalinkId}`}
+                                      href={`/data-tables/permalink/${permalinkId}`}
```

Comment:
any reason this cant be `~/data-tables/permalink` ? would that not be safer

Reply:
Hey @jamescutts looks like use of the tilde in URLs is generally discouraged.